src/badges/stories/index.stories.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
+import faker from '@faker-js/faker'
```

Comment:
We recommend to use 

```ts
import { faker } from '@faker-js/faker'
```

because we want to get rid of the default export in a future major release

---

It is also recommended to use 

```ts
import faker from '@faker-js/faker/locale/en'
```

if you are sure that you only rely on english locales, due to the `import { faker } from '@faker-js/faker'` import is an instance that loads all 56 current availabled languages.
So you would load around 600kb instead of ~2.5mb.

Reply:
Hey @Shinigami92 - I made these suggested changes but it looks like the named import syntax (`import { faker } from '..'`) isn't working (unless it's not meant to be used from a specific locale module like this:)

```js
import { faker } from '@faker-js/faker/locale/en'
```

Reply:
No, you confused it 😅
You should do `import { faker } from '@faker-js/faker'` or `import faker from '@faker-js/faker/locale/en'`

The locale are predefined instances that should be imported with the default import and otherwise you want to get the faker instance with all locales (56) from the index.

Currently we do this to be as compatible as possible to 5.5.3, but in future major versions at least I want to drastically change it. As you see it's just confusing and not so much helpful 🤷 but we are a team of maintainers and they have other opinions about that.

Reply:
For the sake of simplifying the import, I will probably opt for the `import { faker } from '@faker-js/faker'` variant - I have a hunch people won't want to or will forget to dive into the specific en locale for new files unless we add tooling to force it. Thanks for the info though!